libs/Microsoft.MixedReality.WebRTC.Unity/Assets/Microsoft.MixedReality.WebRTC.Unity/Scripts/Media/AudioReceiver.cs

```diff
@@ -144,15 +144,18 @@ internal void AttachToTransceiver(Transceiver audioTransceiver)
         /// with this receiver, which enqueues the <see cref="AudioSource.AudioStreamStarted"/>
         /// event to be fired from the main Unity app thread.
         /// </summary>
-        internal void OnPaired(RemoteAudioTrack track)
+        internal override void OnPaired(MediaTrack track)
         {
+            Debug.Assert(track is RemoteAudioTrack);
+            var remoteAudioTrack = (RemoteAudioTrack)track;
```

Comment:
Do we need the assert? Won't it throw an exception just like the cast?

Reply:
Alright my bad I still had the C/C++ cast model in mind, C# cast actually throws `InvalidCastException`. I will remove the assert.

Reply:
Fixed.